src/components/Content/ContentDetailSmall.vue

```diff
@@ -27,6 +42,62 @@ export default {
     changeStatus(status) {
       // eslint-disable-next-line
       console.log("wanted status", status);
+      axios.get('/api/wanted-list', {
+        headers: {
+          'x-access-token': window.localStorage.getItem('x-access-token'),
+        },
+      }).then((res) => {
+        const wanted = res.data.wantedList.find((wantedList) => wantedList.content_id
+        === this.$props.content.id);
```

Comment:
`this.$props.content` demene gerek yok zaten yukarıda `props` içinde `content` kullanılacağını söyledin.Doğrudan `this.content` kullanabilirsin